templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+    <hr class="p-separator" />
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <!-- rtp section left start -->
+      <div class='p-heading-icon--muted'>
+        <div class='p-heading-icon__header'>
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg",
+              alt="",
+              height="28",
+              width="32",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class='p-heading-icon__title'>Press release</h4>
+        </div>
+      </div>
+      <p>Bosch Rexroth&rsquo;s CtrlX automation platform, built on Ubuntu Core, reduces engineering costs by 30% to 50%.</p>
+      <p><a href='https://ubuntu.com/blog/bosch-rexroth-adopts-ubuntu-core-and-snaps-for-app-based-ctrlx-automation-platform' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'core - left', 'eventLabel' : 'Case study - IMS Evolve ', 'eventValue' : '1' });'>Bosch Rexroth adopts Ubuntu&nbsp;&rsaquo;</a></p>
+      <!-- rtp section left end -->
+    </div>
+    <div class="col-4 p-divider__block">
+      <!-- rtp section right start -->
+      <div class='p-heading-icon--muted'>
+        <div class='p-heading-icon__header'>
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/f6d6f78b-Case+study.svg",
+              alt="",
+              height="28",
+              width="32",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class='p-heading-icon__title'>Case study</h4>
+        </div>
+      </div>
+      <p>Ubuntu Core makes the Dell Edge Gateways ultra reliable in challenging industrial settings with transactional software updates.</p>
+      <p><a href='https://ubuntu.com/blog/the-dell-edge-gateway-3000-launches-with-ubuntu-core-16' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'core - left', 'eventLabel' : 'Case study - IMS Evolve ', 'eventValue' : '1' });'>Dell gateway launches with Ubuntu&nbsp;&rsaquo;</a></p>
+      <!-- rtp section right end -->
+    </div>
+    <div class="col-4 p-divider__block">
+      <!-- rtp section right start -->
+      <div class='p-heading-icon--muted'>
+        <div class='p-heading-icon__header'>
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/f6d6f78b-Case+study.svg",
+              alt="",
+              height="28",
+              width="32",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class='p-heading-icon__title'>Case study</h4>
```

Comment:
You seem to be mixing single quotes with double-quotes. Could you go for double-quotes for consistency please?